docker-compose.override.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
```

Comment:
> Just to let you know, you shouldn't ever need to commit an override file for testing purposes and you only need to replace the keys that have changed from the main `docker-compose.yml`. That is, the `labels` could likely be omitted.

Okay, thanks. 



> Here's a quick summary of the tests I've been doing:
> 
> I've spun up an orchestrator instance and an openstreetmap instance on Unicorn on their own. I'll then send the contents of the JSON file above to the orchestrator under the `/render/preprocess` path. This ends up looking like `sendiagereq $PATH_TO_JSON http://$ORCHESTRATOR_IP:8080/render/preprocess` or equivalently `curl -H "Content-Type: application/json" -d@$PATH_TO_JSON http://$ORCHESTRATOR_IP:8080/render/preprocess`. I monitor the logs of these containers using `docker-compose logs -f`. The 15 s timeout is a limit enforced by the orchestrator to prevent preprocessors from delaying the message. Timeouts between the openstreetmap container and the Overpass API server only happened once with a gateway timeout error returned by the preprocessor _after_ the orchestrator had already timed out waiting on the preprocessor.

I stand to be corrected. It looks  to me like I'm employing an entirely different approach for the POST request, on my local terminal using Git Bash. On the unicorn server, I encountered an error when running  `docker-compose up -d orchestrator openstreetmap`.  I precisely got this: 
"ERROR: for orchestrator  Cannot start service orchestrator: Unable to find group
ERROR: Encountered errors while bringing up the project."
Do I need permissions to run it? I guess I would need the orchestrator to work well before I can access it for a request via ` /render/preprocess`.

More also, it seems to me that the schema for the openstreetmap is yet to be merged to the main, or is there any thing I'm not getting well? I received this error: "
 File "/app/main.py", line 34, in get_map_data
    with open('./schemas/preprocessors/openstreetmap.schema.json') as jsonfile:
FileNotFoundError: [Errno 2] No such file or directory: './schemas/preprocessors/openstreetmap.schema.json'
^C[2022-06-15 12:16:11 +0000] [1] [INFO] Handling signal: int
[2022-06-15 12:16:11 +0000] [7] [INFO] Worker exiting (pid: 7)
[2022-06-15 12:16:11 +0000] [1] [INFO] Shutting down: Master"

The above error came after I sent the request below directly on the unicorn server:
``` curl -X POST \
  http://localhost:5000/preprocessor \
  -H 'cache-control: no-cache' \
  -H 'content-type: application/json' \
  -H 'postman-token: 0bc130ca-ab92-ba34-bb65-603e5b0acb82' \
  -d '{
  "request_uuid": "a3d7b6be-8f3b-47ba-8b17-4b0557e7a8ce",
  "timestamp": 1637789517,
  "coordinates": {
      "latitude": 47.620452,
      "longitude": -122.348793
  },
  "context": "",
  "language": "en",
  "url": "https://fake.site.com/some-url",
  "capabilities": [],
  "renderers": [
    "ca.mcgill.a11y.image.renderer.Text",
    "ca.mcgill.a11y.image.renderer.SimpleAudio",
    "ca.mcgill.a11y.image.renderer.SegmentAudio"
  ]
}'

```

Reply:
You would need to update the submodule to current schemas. Main is currently set up to use a slightly older version. It should be updated as part of your PR. Rebuilding the openstreetmap container should fix this. You can use the remote version of the orchestrator as well.

As for the group error with the orchestrator, you need to specify the docker group ID to it in the environment variable `DOCKER_GID`. On Unicorn, this would be 134, but note that this value will change from system to system.